superset/assets/cypress/integration/dashboard/dashboard_tests.js

```diff
@@ -0,0 +1,26 @@
+describe('Load dashboard', function () {
+  it('Load birth names dashboard', function () {
+    cy.server();
+    cy.login();
+
+    cy.visit('/superset/dashboard/births');
+
+    cy.route('POST', '/superset/explore_json/**').as('getJson');
+    cy.wait(6000, ['@getJson']);
```

Comment:
Setting a specific time to wait is usually not necessary, but there's currently an open issue to better handle multiple matched responses. For now this is the workaround: https://github.com/cypress-io/cypress/issues/477